### PR TITLE
feat(core): experimental features ui

### PR DESCRIPTION
**Pull request by pengx17**
fix TOV-280

experimental features ui
- enabled in the workspace settings for a cloud workspace; only show for workspace owner + early access
- a disclaimer prompt will be shown before going to the next feature setting page
- for now only show the ai poc switch, which controls the display of the ai tab in editor's sidepanel

### CODE REVIEW

**Comment by pengx17** [comment]
Current dependencies on/for this PR:
* `canary`
  * **PR #5337** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5337?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #5338** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5338?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5338?utm_source=stack-comment).

**Review by darkskygit — APPROVED** [review]

File: packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx [hunk 1]
```diff
@@ -205,48 +210,106 @@ export const WorkspaceList = ({
...
+    currentWorkspace.flavour,
+    isOwner,
+    onClick,
+    t,
+  ]);
+
+  return (
+    <>
+      <div
+        className={clsx(style.sidebarSelectItem, { active: !!activeSubTab })}
```
Comment (joooye34) [inline packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx:288]: image

Is the active style of sub table match expectation? @hwangdev97

File: packages/frontend/core/src/components/affine/setting-modal/workspace-setting/experimental-features/arts.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import LightSvg from './light-art-svg';
+
+export const ExperimentalFeatureArts = () => {
+  const { resolvedTheme } = useTheme();
+
+  return (
+    <div
+      className={styles.root}
+      dangerouslySetInnerHTML={{
+        __html: resolvedTheme === 'dark' ? DarkSvg : LightSvg,
```
Comment (joooye34) [inline packages/frontend/core/src/components/affine/setting-modal/workspace-setting/experimental-features/arts.tsx:14]: Maybe use a function to calculate the string?

Comment (pengx17) [reply packages/frontend/core/src/components/affine/setting-modal/workspace-setting/experimental-features/arts.tsx:14]: Do you mean using a template to generate dark/light svg? I compared the two svgs and there seem to be many but subtle differences. I guess it is not worth doing so.

**Comment by hwangdev97** [comment]
> 
> 
> image
> 
> Is the active style of sub table match expectation? @hwangdev97 
> 
> Maybe use a function to calculate the string?

**Comment by hwangdev97** [comment]
> > 
> > 
> > image
> > 
> > Is the active style of sub table match expectation? @hwangdev97 
> > 
> > Maybe use a function to calculate the string?
> 
> I checked the figma file, I think it is right.

File: packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx (hunk 1 above)
Comment (hwangdev97) [reply packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx:288]: yup, I think the active color should be `textPrimary`

Comment (hwangdev97) [reply packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx:288]: It already is textPrimary. I think it's 👌

**Pull request merged**